src/Runner.java

```diff
@@ -0,0 +1,82 @@
+import templates.Account;
+import templates.Blockchain;
+import templates.Miner;
+import utils.SerializationUtil;
+
+import java.io.File;
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+
+public class Runner {
+    public static void main(String[] args) throws InterruptedException, NoSuchAlgorithmException {
+        String path = "blockchain.txt";
+        File file = new File(path);
+        if (file.exists() && !file.isDirectory()) {
+            try {
+                Blockchain importedBlockChain = (Blockchain) SerializationUtil.deserialize(path);
+                Blockchain.getInstance().setBlocksList(importedBlockChain.getBlocksList());
+            } catch (IOException e) {
+                e.printStackTrace();
+            } catch (Exception e) {
+                e.printStackTrace();
+                return;
+            }
+        }
+
+
+        List<Account> accounts = List.of(
```

Comment:
if we want to pass this values passed as mocked values, this can be extracted to some separate generator or some config/file with initial/mock values